specification/appconfiguration/resource-manager/readme.md

```diff
@@ -26,16 +26,25 @@ These are the global settings for the AppConfiguration API.
 
 ``` yaml
 openapi-type: arm
-
-tag: package-2023-03-01
+tag: package-preview-2023-08
 ```
 
 
+### Tag: package-preview-2023-08
+
+These settings apply only when `--tag=package-preview-2023-08` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2023-08'
+input-file:
```

Comment:
Why does this new tag include three version of swagger files? This causes `DuplicateSchema` issue when generating SDK.

Reply:
When working on these changes in the private repo, we had seen Swagger Avacado errors that guided us towards adding the two older swagger files.

Is there a different way to do this that is compatible with SDK generation?

Note: The new API version in this PR didn't remove the paths. The prior version (2023-03-01) did not have those paths either.

Rule | Message
-- | --
❌ MISSING_APIS_IN_DEFAULT_TAG | The   default tag should contain all APIs. The API path /subscriptions/{}/resourceGroups/{}/providers/Microsoft.AppConfiguration/configurationStores/{}/ListKeys is not in the default tag.   Please make sure the missing API swaggers are in the default tag.   readme: specification/appconfiguration/resource-manager/readme.md   json: Microsoft.AppConfiguration/preview/2019-11-01-preview/appconfiguration.json
❌ MISSING_APIS_IN_DEFAULT_TAG | The   default tag should contain all APIs. The API path /subscriptions/{}/resourceGroups/{}/providers/Microsoft.AppConfiguration/configurationStores/{}/RegenerateKey is not in the default tag.   Please make sure the missing API swaggers are in the default tag.   readme: specification/appconfiguration/resource-manager/readme.md   json: Microsoft.AppConfiguration/preview/2019-11-01-preview/appconfiguration.json
❌ MISSING_APIS_IN_DEFAULT_TAG | The   default tag should contain all APIs. The API path /subscriptions/{}/resourceGroups/{}/providers/Microsoft.AppConfiguration/configurationStores/{}/listKeyValue is not in the default tag.   Please make sure the missing API swaggers are in the default tag.   readme: specification/appconfiguration/resource-manager/readme.md   json: Microsoft.AppConfiguration/preview/2019-11-01-preview/appconfiguration.json
❌ MISSING_APIS_IN_DEFAULT_TAG | The   default tag should contain all APIs. The API path /subscriptions/{}/resourceGroups/{}/providers/Microsoft.AppConfiguration/configurationStores/{}/keyValues is not in the default tag.   Please make sure the missing API swaggers are in the default tag.   readme: specification/appconfiguration/resource-manager/readme.md   json: Microsoft.AppConfiguration/preview/2022-03-01-preview/appconfiguration.json

Reply:
Tried to remove the older API swagger files and see that Avacado is no longer required. The prior API version 2023-03-01 didn't have the older API swagger files either so leaving them out to fix the required check